COVERAGE.md

```
@@ -740,302 +295,160 @@
-All files          |     100 |      100 |     100 |     100 |                  
- bundler.js        |     100 |      100 |     100 |     100 |                  
- ...ontractPath.js |     100 |      100 |     100 |     100 |                  
- index.js          |     100 |      100 |     100 |     100 |                  
- readCache.js      |     100 |      100 |     100 |     100 |                  
- readCacheSync.js  |     100 |      100 |     100 |     100 |                  
- ...ModuleAsync.js |     100 |      100 |     100 |     100 |                  
- ...eModuleSync.js |     100 |      100 |     100 |     100 |                  
- types.ts          |       0 |        0 |       0 |       0 |                  
- writeCache.js     |     100 |      100 |     100 |     100 |                  
- writeCacheSync.js |     100 |      100 |     100 |     100 |                  
+All files          |   31.28 |    95.54 |   92.85 |   31.28 |                   
```

markdownlint-cli2 (0.17.2)

Table pipe style
Expected: no_leading_or_trailing; Actual: trailing_only; Unexpected trailing pipe

(MD055, table-pipe-style)